apps/docs/src/content/reference/extras/instanced-sprite.mdx

```diff
@@ -0,0 +1,454 @@
+            type: 'Vector3Tuple',
+            required: false,
+            description: 'Sets the position of an instance.'
+          },
+					{
+            name: 'scale',
+            type: 'Vector2Tuple',
+            required: false,
+            description: 'Sets the scale of an instance.'
+          },
+					{
+            name: 'playmode',
+            type: '"FORWARD" | "REVERSE" | "PAUSE" | "PINGPONG"',
+            required: false,
+            description: 'Sets the playmode of an instance.'
+          },
+					{
+            name: 'billboarding',
+            type: 'boolean',
+            required: false,
+            description: 'Toggles billboarding on/off.'
+          },
+					{
+            name: 'offset',
+            type: 'number',
+            required: false,
+            description: 'Sets the exact time value in ms by which the animation playback is offset.'
+          },
+					{
+            name: 'loop',
+            type: 'boolean',
+            required: false,
+            description: 'Toggles looping of the animation on/off. If off, the last played frame is displayed when the animation completes.'
+          },
+					{
+            name: 'flipX',
+            type: 'boolean',
+            required: false,
+            description: 'Toggles flipping the sprite horizontally.'
+          },
+					{
+            name: 'flipY',
+            type: 'boolean',
+            required: false,
+            description: 'Toggles flipping the sprite vertically.'
+          },
+					{
+            name: 'frameId',
+            type: 'number',
+            required: false,
+            description: `Sets an exact frame to display. If animationName is set, it's the id of the frame in that given animation, otherwise it's a spritesheet frame id.`
+          }
+        ]
+    }}
+	/>
+
+
+### binding
+
+The `InstancedSpriteMesh` class is the foundation behind the `<InstancedSprite>` component, written to enable efficient instancing of animated sprites within a
+Three.js environment. The `<InstancedSprite>` component lets you bind to it through `ref`.
+
+The class extends the capabilities of the [troika's InstancedUniformsMesh](https://protectwise.github.io/troika/three-instanced-uniforms-mesh/). For an in-depth exploration of `InstancedSpriteMesh` and its features, refer to the documentation available at [InstancedSpriteMesh docs](https://three-kit.vercel.app/instancedsprite/01-instanced-sprite-mesh/).
+
+
+## Spritesheets
+
+### SpritesheetMetadata
+
+Object used in `buildSpritesheet` function has to be compliant with the `SpritesheetMetadata` type format. This type is structured to
+accommodate the metadata for one or multiple sprite files within a single spritesheet.
+
+```ts
+type SpritesheetMetadata = {
+  url: string
+  type: 'rowColumn' | 'frameSize'
+  width: number
+  height: number
+  animations: {
+    name: string
+    frameRange: [number, number]
+  }[]
+}[]
+
+```
+
+#### Understanding SpritesheetMetadata
```

Comment:
Remove this heading; the contents of the subsection flow well with the previous content, so its not needed.